src/what_is_an_ABI.md

```diff
@@ -32,3 +32,13 @@ To quote from the initial proposal:
 > TODO: more recent examples.
 
 ## What would a stable modular ABI look like for Rust?
+
+- **Fully specified** - There should be no ambiguous cases, either in how to map Rust to the ABI, or how the ABI maps back to Rust objects.
```

Comment:
Some things that still needs some thought (and this page might not be the best page to address it) are pointers (specifically smart pointers, like Rc, etc.), how to specify calling convention, and move semantics.

Reply:
I agree.  The main goal for starting that list was to build a checklist of what we need.  Basically, I want to know when we're done.  I also want to know what's out of scope for the project.

Hmmm... maybe that whole list would be better off on another page; 'Goals'?  But it needs to be a real checklist, one where we can create action items, create issues and projects around the items, and make decisions.  What I've got is pretty ambiguous, it was just meant to start the ball rolling.